Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -5411,186 +5444,1331 @@ def panorama_show_location_ip_command():
+                   '<size-limit>1000</size-limit></entry><entry name="ms-office"><size-limit>2000</size-limit></entry>'
+                   '<entry name="jar"><size-limit>5</size-limit></entry><entry name="flash">'
+                   '<size-limit>5</size-limit></entry><entry name="MacOSX"><size-limit>1</size-limit></entry>'
+                   '<entry name="archive"><size-limit>10</size-limit></entry><entry name="linux">'
+                   '<size-limit>2</size-limit></entry><entry name="script"><size-limit>20</size-limit></entry>'
+                   '</file-size-limit><report-grayware-file>yes</report-grayware-file></wildfire>'
+    }
+    result = http_request(URL, 'POST', params=params)
+
+    return result
+
+
+def enforce_wildfire_schedule(template):
+    params = {
+        'action': 'set',
+        'type': 'config',
+        'xpath': f"/config/devices/entry[@name='localhost.localdomain']/template/entry[@name='{template}']/config/"
+        f"devices/entry[@name='localhost.localdomain']/deviceconfig/system/update-schedule/wildfire",
+        'key': API_KEY,
+        'element': '<recurring><every-min><action>download-and-install</action></every-min></recurring>'
+    }
+
+    result = http_request(URL, 'POST', params=params)
+
+    return result
+
+
+def enforce_wildfire_system_config_command():
+
+    template = demisto.args().get('template')
+    enforce_wildfire_system_config(template)
+    demisto.results('WildFire file upload for all file types is set to the maximum size.')
+
+
+def enforce_wildfire_schedule_command():
+    template = demisto.args().get('template')
+    enforce_wildfire_schedule(template)
+    demisto.results('The schedule was updated according to the best practice.'
+                    '\nRecurring every minute with the action of "download and install"')
+
+
+def url_filtering_block_default_categories(profile_name: str):
+
+    params = {
+        'action': 'set',
+        'type': 'config',
+        'xpath': f"{XPATH_RULEBASE}profiles/url-filtering/entry[@name='{profile_name}']/block",
+        'key': API_KEY,
+        'element': '<member>adult</member><member>hacking</member><member>command-and-control</member><member>'
+                   'copyright-infringement</member><member>extremism</member><member>malware</member><member>'
+                   'phishing</member><member>proxy-avoidance-and-anonymizers</member><member>parked</member><member>'
+                   'unknown</member><member>dynamic-dns</member>'
+    }
+    result = http_request(URL, 'POST', params=params)
+
+    return result
+
+
+def url_filtering_block_default_categories_command():
+
+    profile_name = demisto.args().get('profile_name')
+    url_filtering_block_default_categories(profile_name)
+    demisto.results(f'The default categories to block has been set successfully to {profile_name}')
+
+
+def get_url_filtering_best_practice_command():
+
+    best_practice = {
```

Comment:
who defined it was best practice? let's comment that.

Reply:
It's Palo Alto best practice recommendation. 
Will add that in readme, after docs review